<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>900</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>732</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_webcam">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>WebCam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stopcam">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>StopCam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Basics</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_resize">
      <property name="geometry">
       <rect>
        <x>137</x>
        <y>9</y>
        <width>141</width>
        <height>122</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Resize</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_resize">
         <property name="text">
          <string>Resize</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_width">
         <property name="text">
          <string>width</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_width">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_height">
         <property name="text">
          <string>height</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_height">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_flip">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>122</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Flip</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_horiz">
         <property name="text">
          <string>Horizontal</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_vert">
         <property name="text">
          <string>Vertical</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_flip">
         <property name="text">
          <string>Flip</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_hist">
      <property name="geometry">
       <rect>
        <x>137</x>
        <y>137</y>
        <width>105</width>
        <height>89</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Histogram</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_eq">
         <property name="text">
          <string>Equalize</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_hist">
         <property name="text">
          <string>Display</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_noise">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>137</y>
        <width>105</width>
        <height>56</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Noise</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_sp">
         <property name="text">
          <string>S and P</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Filters</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_kernel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Kernel</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Erode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dilate</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Open</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Close</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_Blur">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Blur</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Edges</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_canny">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Canny</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_lapl">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Laplacian</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sobel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Sobel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Features</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_harris">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Harris</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_surf">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>SURF</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Detectors</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Descriptors</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sift">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>SIFT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_fast">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>FAST</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Others</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_logo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Logo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_invert">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Invert</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_circles">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Circles</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_lines">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Lines</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_histR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>211</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{border: 1px solid gray;
    border-radius: 5px;
    margin-top: 0.5em;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_histG">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>530</y>
      <width>211</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{border: 1px solid gray;
    border-radius: 5px;
    margin-top: 0.5em;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_histB">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>530</y>
      <width>211</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{border: 1px solid gray;
    border-radius: 5px;
    margin-top: 0.5em;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
